packages/nuxt/src/app/nuxt.ts

```diff
@@ -103,7 +103,7 @@ interface _NuxtApp {
   [key: string]: unknown
 
   /** @internal */
-  _asyncDataPromises: Record<string, Promise<any> | undefined>
+  _asyncDataPromises: Record<string, Promise<any> & { strategy?: 'lazy' | 'blocking' | 'parallel' } | undefined>
```

Comment:
Small suggestion, this union is duplicated with `AsyncDataOptions` - might be worth to extract both to a type, or get it from the options interface:

```suggestion
  _asyncDataPromises: Record<string, Promise<any> & AsyncDataOptions['strategy']>
```

Reply:
We might also make a type export for `AsyncDataStrategy` 👍🏼